cache/image_cache.go

```diff
@@ -22,18 +23,19 @@ type ImageCache struct {
 	newImage  imgutil.Image
 }
 
-func NewImageCache(origImage imgutil.Image, newImage imgutil.Image) *ImageCache {
+func NewImageCache(origImage imgutil.Image, newImage imgutil.Image) (*ImageCache, error) {
 	return &ImageCache{
 		origImage: origImage,
 		newImage:  newImage,
-	}
+	}, nil
```

Comment:
Why is this needed? Couldn't we just keep doing `return NewImageCache(origImage, emptyImage), nil`?

Reply:
Oh, good catch. I'll fix this. I removed the section that threw the error and forgot to clean this up.